modules/gcp_bigquery_table/main.tf

```diff
@@ -4,25 +4,23 @@ terraform {
       source  = "hashicorp/google"
       version = "~> 4.13.0"
```

Comment:
Component modules should use `>=`, layer module providers use `~>` upto the minor version (eg `~> 4.13`).

Reply:
Done